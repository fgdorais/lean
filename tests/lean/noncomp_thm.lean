noncomputable theorem foo : true := trivial
