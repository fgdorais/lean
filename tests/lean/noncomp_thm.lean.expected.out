noncomp_thm.lean:1:0: error: invalid 'noncomputable' declaration, it can only be used for definitions
